server/src/main/java/org/apache/iotdb/db/metadata/MetadataConstant.java

```diff
@@ -40,7 +40,28 @@ private MetadataConstant() {
   public static final String MTREE_SNAPSHOT_TMP =
       MTREE_PREFIX + IoTDBConstant.FILE_NAME_SEPARATOR + MTREE_VERSION + ".snapshot.bin.tmp";
 
-  public static final short MNODE_TYPE = 0;
-  public static final short STORAGE_GROUP_MNODE_TYPE = 1;
-  public static final short MEASUREMENT_MNODE_TYPE = 2;
+  public static final byte INTERNAL_MNODE_TYPE = 0;
+  public static final byte STORAGE_GROUP_MNODE_TYPE = 1;
+  public static final byte MEASUREMENT_MNODE_TYPE = 2;
+  public static final byte ENTITY_MNODE_TYPE = 3;
```

Comment:
How to upgrade from 0.12?

Reply:
Currently, they are only used in MNodeTypeMismatchException and there are no problem with upgrade.